src/audio/rsp_mixer.S

```diff
@@ -276,9 +276,9 @@ command_exec:
 	#define samples_left    t4
 	#define outptr          s8
 
-	vxor v_zero, v_zero, v_zero,0
+	vxor v_zero, v_zero, v_zero
 	li t0, %lo(VCONST_1)
-	lqv v_const1,0, 0,t0
+	lqv v_const1,e(0), 0,t0
```

Comment:
Honestly I don’t know if I’d made these changes. It makes the code more verbose and honestly the real solution here would be making that argument optional. If that’s not easy to do, maybe just leave 0?

Reply:
Yeah you're right. I'll look into this again. I wonder if there is a good, general approach to providing multiple "overloads" of GAS macros

Reply:
The element is now optional